src/issues/issueHoverProvider.ts

```diff
@@ -12,6 +12,10 @@ export class IssueHoverProvider implements vscode.HoverProvider {
 	constructor(private manager: PullRequestManager, private stateManager: StateManager) { }
 
 	provideHover(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken): vscode.ProviderResult<vscode.Hover | undefined> {
+		if (document.lineAt(position.line).range.end.character > 300) {

```

Comment:
should this be changed to 10000 per https://github.com/microsoft/vscode-pull-request-github/issues/1629#issuecomment-613991938?